Cookies from browser (#488)
yt-dlp/yt-dlp: README.md

* [**mutagen**](https://github.com/quodlibet/mutagen) - For embedding thumbnail in certain formats. Licenced under [GPLv2+](https://github.com/quodlibet/mutagen/blob/master/COPYING)
* [**pycryptodome**](https://github.com/Legrandin/pycryptodome) - For decrypting various data. Licenced under [BSD2](https://github.com/Legrandin/pycryptodome/blob/master/LICENSE.rst)
* [**websockets**](https://github.com/aaugustin/websockets) - For downloading over websocket. Licenced under [BSD3](https://github.com/aaugustin/websockets/blob/main/LICENSE)

> * [**keyring**](https://github.com/jaraco/keyring) - For decrypting chromium cookies on some UNIX systems. Licenced under [MIT](https://github.com/jaraco/keyring/blob/main/LICENSE)

Reviewer: Can you confirm `For decrypting chromium cookies on some UNIX systems` is an accurate description of keyring's use for us?

Author: more or less. It can be used on OSX but calling out to a built-in command works just as well, so it's only necessary on Linux. I think I would go ahead and just say `for decrypting cookies of chromium-based browsers on Linux` because as far as I know chromium doesn't officially support any other UNIX-like OS like BSD

Reviewer: ```suggestion
* [**keyring**](https://github.com/jaraco/keyring) - For decrypting cookies of chromium-based browsers on Linux. Licenced under [MIT](https://github.com/jaraco/keyring/blob/main/LICENSE)
```
